WebRoot/Templates/navbar admin.php

```diff
@@ -72,6 +82,6 @@
 		</div>
 	</section>
 	<?php endif ?>
-	<div id="version">Wuwana version 2.2.1</div>
+	<!-- <div id="version">Wuwana version 2.2.1</div> -->
```

Comment:
You can remove this line completely now we have this info in the "Dashboard" page (after login) in `WebRoot/admin/view.php`